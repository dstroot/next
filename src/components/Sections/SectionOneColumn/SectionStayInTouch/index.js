import React from 'react';

//components
import SectionOneColumn from '..';
import Button from '../../../Button';

//animation
// import Fade from 'react-reveal/Fade';

const SectionStayInTouch = () => {
  return (
    <SectionOneColumn
      heading="Stay in Touch"
      content={
        <div>
          <p className="mb-4">Enter your email address to receive updates</p>
          <div class="container">
            <div class="row justify-content-md-center mb-4">
              <div class="col-md-6">
                <input
                  className="form-control"
                  type="text"
                  placeholder="Email Address"
                  id="InputEmail"
                />
              </div>
            </div>
          </div>
          <Button path="/" buttonText="Submit" />
        </div>
      }
    />
  );
};

export default SectionStayInTouch;
