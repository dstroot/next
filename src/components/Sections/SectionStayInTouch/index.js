import React from 'react';

//components
import Button from '../../Button';

//animation
import Fade from 'react-reveal/Fade';

const SectionStayInTouch = () => {
  return (
    <div className="py-6">
      <div className="container text-center">
        <h1 className="nxt_heading-large mb-5">Stay in Touch</h1>
        <Fade bottom>
          <p>Enter your email address to receive updates</p>
        </Fade>
        <Button path="/" buttonText="Submit">
          Sup
        </Button>
      </div>
    </div>
  );
};

export default SectionStayInTouch;
