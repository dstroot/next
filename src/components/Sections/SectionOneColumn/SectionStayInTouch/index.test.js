import React from 'react';
import { create } from 'react-test-renderer';
import { mount } from 'enzyme';
import SectionStayInTouch from '../SectionStayInTouch';

describe('SectionStayInTouch', () => {
  // mount the component
  let mountedComponent;
  const getMountedComponent = () => {
    if (!mountedComponent) {
      mountedComponent = mount(<SectionStayInTouch />);
    }
    return mountedComponent;
  };

  beforeEach(() => {
    mountedComponent = undefined;
  });

  it('it should render', () => {
    let tree = create(<SectionStayInTouch />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
