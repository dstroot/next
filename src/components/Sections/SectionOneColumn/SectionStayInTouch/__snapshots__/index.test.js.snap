// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionStayInTouch it should render 1`] = `
<div
  id="stayintouch"
>
  <div
    className="py-6 undefined"
  >
    <div
      className="container text-center"
    >
      <div
        className="react-reveal nxt_heading-xlarge mb-5"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        Stay in Touch
      </div>
      <div
        className="react-reveal"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <div>
          <p
            className="mb-4"
          >
            Enter your email address to receive updates
          </p>
          <form
            onSubmit={[Function]}
          >
            <div
              className="container"
            >
              <div
                className="row justify-content-md-center mb-4"
              >
                <div
                  className="col-md-6"
                >
                  <input
                    className="form-control"
                    name="email"
                    onChange={[Function]}
                    placeholder="Email Address"
                    required={true}
                    type="email"
                    value=""
                  />
                </div>
              </div>
            </div>
            <button
              className="btn btn-primary rounded-pill nxt_body-small px-4 py-2 submit"
            >
              Submit
            </button>
          </form>
          <h4
            className="react-reveal alert col-md-6"
            style={
              Object {
                "opacity": undefined,
              }
            }
          >
            You are on the list
          </h4>
        </div>
      </div>
      <div />
    </div>
  </div>
</div>
`;
